### DISCUSSION TITLE
Spades for larger genomes

### DISCUSSION THREAD

**Discussion by eleniadam** [ablab/spades#884] [Q&A]
Hello,

In the manual, it is stated that: "SPAdes is not intended for larger genomes (e.g. mammalian size genomes). For such purposes you can use it at your own risk.". Could you please explain to me what exactly may cause a problem with using larger genomes in Spades?

Will Spades work and compute a correct assembly but it will take a longer time? Or is there a specific reason causing the algorithm to crash or compute an incorrect assembly with larger genomes?

I also wanted to ask if there will be a parallel/GPU-accelerated version of Spades assembly in the near future? (Which could potentially help speed up the calculation for larger genomes... - if that is an issue).

Thank you,
Eleni

**Comment by asl** [comment]
Hello

> Could you please explain to me what exactly may cause a problem with using larger genomes in Spades?

Well, the reason is simple: we did not do any extensive testing of SPAdes for large genomes, so some problems might be there. Also, it does not make much sense to assemble them using short Illumina reads as they are quite complex repeat-wise.

Certainly assembling large genomes might take quite some time and computational resources. However, users used SPAdes recently on very large soil metagenomes w/o much troubles.

> I also wanted to ask if there will be a parallel/GPU-accelerated version of Spades assembly in the near future? 

GPUs are pretty useless for genome assembly tasks. And SPAdes itself could utilize multiple CPU cores when possible,
